src/vs/editor/contrib/referenceSearch/browser/referencesWidget.ts

```diff
@@ -479,12 +496,19 @@ export interface SelectionEvent {
 }
 
 /**
+ * Map of FileReferences by id.
+ */
+type FileReferenceSet = Map<string, FileReferences>;
+
+/**
  * ZoneWidget that is shown inside the editor
  */
 export class ReferenceWidget extends PeekViewWidget {
 
 	private _model: ReferencesModel;
 	private _decorationsManager: DecorationsManager;
+	private _expandedElements: FileReferenceSet = new Map<string, FileReferences>();
```

Comment:
I don't think the type alias here has any effect. IntelliSense will resolve type aliases to the concrete type. I would just omit the type declaration here, and move above comment to the property here.